Sources/Orders/DTOs/OrderJSON.swift

```diff
@@ -68,20 +102,12 @@ extension OrderJSON {
     public enum SchemaVersion: Int, Encodable {
         case v1 = 1
     }
-}
 
-extension OrderJSON {
-    /// A protocol that represents the merchant associated with the order.
-    ///
-    /// > Tip: See the [`Order.Merchant`](https://developer.apple.com/documentation/walletorders/merchant) object to understand the keys.
-    public protocol Merchant: Encodable {
-        /// The localized display name of the merchant.
-        var displayName: String { get }
-        
-        /// The Apple Merchant Identifier for this merchant, generated at `developer.apple.com`.
-        var merchantIdentifier: String { get }
-        
-        /// The URL for the merchant’s website or landing page.
-        var url: String { get }
+    /// The format of the barcode.
+    public enum BarcodeFormat: String, Encodable {
```

Comment:
Are we sure this won't get any new cases in the future? Because that will introduce breaking changes

Reply:
Realistically: no. But I could hide it behind a struct just in case

Reply:
If I change `BarcodeFormat` to a `struct`, how can I pass the raw values for encoding? `public static let` variables don't seem to work too well...

Reply:
You can make `public static func`s

Reply:
The fact is that right now `BarcodeFormat` (and many other enums) are used in protocols, like this:
```swift
public struct OrderJSON {
    ...

    public protocol Barcode: Encodable {
        var format: BarcodeFormat { get }
        ...
    }

    public enum BarcodeFormat: String, Encodable {
        case pdf417
        case qr
        ...
    }
}
```

So a struct that conforms to `Barcode` looks like this:
```swift
struct Foo: OrderJSON.Barcode {
    let format = OrderJSON.BarcodeFormat.qr
    ...
}
```

If I change `BarcodeFormat` to a struct, it won't support enums raw value, so the protocol variable would have to be a simple `String`, allowing any unsupported JSON key to be written. Any other abstraction (that limits to the supported keys) that I can think of requires the enum to be public.

Does my reasoning make sense or am I missing something?

Reply:
Back from holiday! Yeah I'd say let's leave it as an enum for now and recheck stuff before v1 drops

Reply:
Thanks Paul, can I ask you then for a sweet sweet green tick on this PR? 😅